### ISSUE TITLE
"Create" and the windows backslash

### ISSUE THREAD

**Issue by hansbkk** [Compass/compass#156]
Bottom line - use forward slashes and all's well.

This syntax
compass create bp --using blueprint --css-dir "htdocs\styles" --sass-dir "compass\sass-src"  --images-dir "htdocs\images"

creates everything OK, except for:
Nothing to compile. If you're trying to start a new project, you have left off the directory argument.

But _does_ put this in the config.rb:
 css_dir = "htdocs\styles"
 sass_dir = "compass\sass-src"
 images_dir = "htdocs\images"
 javascripts_dir = "htdocs\scripts"

Using a second backslash to escape:
compass create bp --using blueprint --css-dir "htdocs\styles" --sass-dir "compass\sass-src"  --images-dir "htdocs\images" --javascripts-dir "htdocs\scripts"

results in this config.rb
 css_dir = "htdocs\\styles"
 sass_dir = "compass\\sass-src"
 images_dir = "htdocs\\images"
 javascripts_dir = "htdocs\\scripts"

And both result in weird output from the "create" command, both at the top when it's reporting on what it's doing, and at the end in the advisory CSS linking code for your HTML.

IMO easiest might be to just catch any backslashed paths and quit with a message to the user telling them to use *nix-style forward slashes, with examples.

Or, if you are going to convert, you should (again IMO) convert everywhere to the canonical format (double quotes and forward slashes?), including both top and bottom of the screen output after Create runs, and the contents of the config.rb.

**Comment by hansbkk** [comment]
sorry didn't use the markdown

Using a second backslash to escape:

```
 compass create bp --using blueprint --css-dir "htdocs\styles" --sass-dir "compass\sass-src" --images-dir "htdocs\images" --javascripts-dir "htdocs\scripts"
```

results in this config.rb

```
 css_dir = "htdocs\\\\styles"
 sass_dir = "compass\\\\sass-src"
 images_dir = "htdocs\\\\images"
 javascripts_dir = "htdocs\\\\scripts"
```

**Comment by hansbkk** [comment]
just noticed that my "second backslash" example above still didn't show the two backslashes, but I'm sure you get the idea

**Comment by chriseppstein** [comment]
Fix confusing error scenario on windows. Closed by f4a1e85f34236d57f6d87231a08c5a1434b513d9.
